controls/database.sp

```diff
@@ -0,0 +1,86 @@
+locals {
+  database_common_tags = merge(local.thrifty_common_tags, {
+    service = "database"
+  })
+}
+
+benchmark "database" {
+  title         = "Oracle Database Checks"
+  description   = "Thrifty developers checks old autonomous database which were created over 90 days ago."
+  documentation = file("./controls/docs/database.md")
+  tags          = local.database_common_tags
+  children = [
+    control.database_autonomous_database_age_90,
+    control.database_autonomous_database_low_utilization
+  ]
+}
+
+control "database_autonomous_database_age_90" {
+  title       = "Autonomous database created over 90 days ago should be reviewed"
```

Comment:
If there aren't any equivalent settings for reserved instances, or another cost saving measure for long running instances, I think we should remove this control. Having an autonomous database running for 90 days ago or more alone isn't enough to recommend a deletion.